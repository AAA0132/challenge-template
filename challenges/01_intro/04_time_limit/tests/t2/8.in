724815575983048299179
