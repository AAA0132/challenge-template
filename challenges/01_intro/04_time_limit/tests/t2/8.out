262678809593819031205753299972353870186610
